pydocstringformatter/_formatting/_utils.py

```diff
@@ -8,3 +12,18 @@ def is_rst_title(summary: str) -> bool:
     if not (last_line := summary.splitlines()[-1].lstrip()):
         return False
     return last_line.count(last_line[0]) == len(last_line)
+
+
+def _compare_formatters(
```

Comment:
Since this module name is private we can make this function itself "non-private".

```suggestion
def compare_formatters(
```